src/web/src/app/space/service.ts

```diff
@@ -53,4 +141,200 @@ export class SpaceService {
 	isInitialized(): Observable<boolean> {
 		return this._isInitialized.asObservable();
 	}
+
+	getList(limit: number, page: number, name: string = ''): Observable<PaginatorApi<Space>> {
+		const grouped = this.spaces.filter(space => space.name.includes(name));
+		const res: PaginatorApi<Space> = {
+			count: grouped.length,
+			results: grouped.slice(limit * page, limit * page + limit),
+		};
+		return of(res)
+			.pipe(delay(2000));
+	}
+
+	getById(spaceId: string) {
+		let space = this.spaces.find(s => s.id === spaceId);
+		return of(space)
+			.pipe(delay(2000));
+	}
+
+	getByKey(spaceKey: string) {
+		let space = this.spaces.find(s => s.key === spaceKey);
+		return of(space)
+			.pipe(delay(2000));
+	}
+
+	getInvitationList(limit: number, page: number, spaceId: string = null): Observable<PaginatorApi<SpaceInvitation>> {
+		const grouped = spaceId ?
+			this.invitations.filter(item => item.spaceId === spaceId) :
+			this.invitations;
+
+		const res: PaginatorApi<SpaceInvitation> = {
+			count: grouped.length,
+			results: grouped.slice(limit * page, limit * page + limit),
+		};
+		return of(res)
+			.pipe(delay(2000));
+	}
+
+	getUserListForSpace(spaceId: string, limit: number, page: number): Observable<PaginatorApi<SpaceUser>> {
```

Comment:
Вот тут ещё давай тоже пожалуйста Space уберём
```suggestion
	getUserList(spaceId: string, limit: number, page: number): Observable<PaginatorApi<SpaceUser>> {
```

Reply:
Готово.